app/controllers/middleware_server_controller.rb

```diff
@@ -45,6 +45,12 @@ class MiddlewareServerController < ApplicationController
                                     :hawk  => N_('Not deploying to Hawkular server'),
                                     :msg   => N_('Deployment initiated for selected server(s)'),
                                     :param => :file
+    },
+    :middleware_add_jdbcdriver  => {:op    => :add_middleware_jdbc_driver,
+                                    :skip  => false,
+                                    :hawk  => N_('Not installing new JDBC Driver to Hawkular server'),
```

Comment:
~~the `:hawk` and `:msg` fields for `middleware_add_deployment` and `middleware_add_jdbcdriver` are not being used, right?~~

I am trying to figure out, how this can work. It contains the whole sentence, but [here](https://github.com/ammendonca/manageiq/blob/ce960011803ac4d7795732cbb20425b30c6b7484/app/controllers/middleware_server_controller.rb#L192) it surrounds the `msg` with another sentence. I guess, that's what the `skip` field is for (?)